### PR TITLE
Update contributing2.md

### PR DESCRIPTION
**Pull request by chris-4444**
Edited and formatted.

Please make sure to check the following boxes creating your pull request, thank you very much!

- [ ] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [ ] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [ ] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [ ] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [ ] Select a beautiful and meaningful teaser image for your blog
- [ ] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

File: docusaurus/docs/contributing/overview.md [hunk 1]
```diff
@@ -5,33 +5,32 @@ title: Contributing to Wechaty
...
 
 ### Not sure where to start?
 
-It's a myth that writing code is the only way to contribute to open source. Wechaty community is open to new ideas and there are so many different ways to make valuable contributions. We have some ideas of how you can get started!
+It is a myth that writing code is the only way to contribute to open source. The Wechaty community is very open to new ideas and there are so many different ways to make valuable contributions. We even have some ideas on how you can get started!
 
 ## Work on the Wechaty framework
 
-The work on Wechaty itself falls into three major areas:
+The work on Wechaty itself falls into three (3) major areas:
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/docs/contributing/overview.md:24]: Maybe we can add another one

```report bugs, suggest enhancements, requests features as Github issues.```

A subtle way of contributing can be by following and sharing  Wechaty content across social platforms like YT, gitter, etc.

Comment (Rohitesh-Kumar-Jain) [reply docusaurus/docs/contributing/overview.md:24]: What say?

File: docusaurus/docs/contributing/overview.md [hunk 2]
```diff
@@ -5,33 +5,32 @@ title: Contributing to Wechaty
...
+Thank you for your interest in contributing to Wechaty.
 
-This document explains the process of contributing to the Wechaty project.
+This post explains the process of contributing to the Wechaty project.
 
-First of all please follow the [Wechaty Community Code of Conduct](code-of-conduct.md) in all your interactions within the project.
+For all interactions within the project, kindly read and abide by the rules in [Wechaty Community Code of Conduct](code-of-conduct.md).
 
-Wechaty is a community that lives on its volunteers. As it keeps growing, we always need more people to help others. You can contribute in many ways, either to the framework itself or in the wider ecosystem.
+Wechaty is a community that lives on its volunteers. As it keeps growing, we always need more people to help the users and the community. You can contribute in many ways, either to the framework itself or in the wider ecosystem.
```
Comment (proudofsimin) [inline docusaurus/docs/contributing/overview.md]: Wechaty is an open source software which is highly relied on volunteers to maintain. As we keep growing, we always welcome more people to join us. Please read through and abide by [Wechaty Community Code of Conduct](code-of-conduct.md).

File: docusaurus/docs/contributing/overview.md [hunk 3]
```diff
@@ -40,127 +39,117 @@ Start from the [writing code docs](coding.md).
 
 ### Writing documentation ✍️
 
-Wechaty's documentation is one of its key strengths. It's informative
-and thorough. You can help to improve the documentation and keep it
-relevant as the framework evolves.
+Wechaty's documentation is one of its key strengths because it is informative
+and thorough. As the framework evolves, you can help to improve the documentation and keep it
+relevant from time to time.
```
Comment (proudofsimin) [inline docusaurus/docs/contributing/overview.md]: Documentation is one of the most important things that introduce Wechaty to general users. If you like teaching and sharing knowledge with other people, you are more than welcome to join our documentation project. Below are some ways you could contribute:

**Review by chris-4444 — COMMENTED** [review]
Included all suggestions, ty @Rohitesh-Kumar-Jain @proudofsimin. 😎

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Thanks for implementing my suggestions :)

**Comment by Rohitesh-Kumar-Jain** [comment]
ping @nibble0101 and @proudofsimin

File: docusaurus/docs/contributing/overview.md [hunk 4]
```diff
@@ -5,162 +5,149 @@ title: Contributing to Wechaty
...
+- Fix typos in the documentation
 - Translate documentation to your local language
-- Write tutorials and blog posts, see more: <https://github.com/wechaty/wechaty.js.org/tree/master/jekyll/_posts>
-- Answer questions on the Wechaty Developers Home or Github issues
+- Write tutorials and blog posts, refer to <https://github.com/wechaty/wechaty.js.org/tree/master/jekyll/_posts>
+- Answer questions on the Wechaty Developers Home or GitHub issues
 - Organize Wechaty meetups or user groups in your area
 
-Contact rui@Wechaty.io to learn more
+For more information, you may contact rui@Wechaty.io.
```
Comment (lijiarui) [inline docusaurus/docs/contributing/overview.md]: Please change the email from `rui@Wechaty.io` to `wechaty@googlegroups.com`

File: docusaurus/docs/contributing/overview.md [hunk 5]
```diff
@@ -5,162 +5,149 @@ title: Contributing to Wechaty
...
 - Translate documentation to your local language
-- Write tutorials and blog posts, see more: <https://github.com/wechaty/wechaty.js.org/tree/master/jekyll/_posts>
-- Answer questions on the Wechaty Developers Home or Github issues
+- Write tutorials and blog posts, refer to <https://github.com/wechaty/wechaty.js.org/tree/master/jekyll/_posts>
+- Answer questions on the Wechaty Developers Home or GitHub issues
 - Organize Wechaty meetups or user groups in your area
+- Another way of contributing can be by following and sharing Wechaty content across social media platforms like YouTube, Gitter, etc.
 
-Contact rui@Wechaty.io to learn more
+For more information, you may contact rui@Wechaty.io.
```
Comment (lijiarui) [inline docusaurus/docs/contributing/overview.md]: Please change the email from `rui@Wechaty.io` to `wechaty@googlegroups.com`

File: docusaurus/docs/contributing/overview.md [hunk 6]
```diff
@@ -5,162 +5,149 @@ title: Contributing to Wechaty
...
 - Join the [Wechaty channel](https://gitter.im/wechaty/wechaty)
   on Gitter.im and answer questions.
-  By explaining Wechaty to other users, you're going to learn a lot about the
+  By explaining Wechaty to other users, you are going to learn a lot about the
   framework yourself.
 
 - Join the [GitHub Discussions](https://github.com/wechaty/wechaty/discussions).
-  This forum is a place for discussing the Wechaty framework and applications and projects that use it. This is also a good place to ask and answer any questions related to installing, using, or
-  contributing to Wechaty.
+  This forum is a place for discussing the Wechaty framework, applications, and projects that use it. This is also a good place to ask and answer any questions related to installing, using, or contributing to Wechaty.
```
Comment (lijiarui) [inline docusaurus/docs/contributing/overview.md:136]: Add mailing list:

```
- Join Wechaty Mailing List: Send an email to wechaty@googlegroups.com, then you are in!

- Join Wechaty WeChat Room: If you want to talk with other developers via WeChat, just scan the [QR Code](https://github.com/wechaty/wechaty#join-us) in WeChat with secret code wechaty, join our Wechaty Developers' Home.


```

File: docusaurus/docs/contributing/overview.md [hunk 7]
```diff
@@ -5,162 +5,149 @@ title: Contributing to Wechaty
...
+- Join the [Google Group](https://groups.google.com/g/wechaty) and answer some questions.
 
 - Blog about Wechaty. We syndicate all the Wechaty blogs we know about on
   the [community blogs](https://wechaty.js.org/blog);
-  if you'd like to see your blog on that page you can [register it here](blog.md).
+  if you'd like to see your blog on that page, you can [register it here](blog.md).
 
-- Contribute to open-source Wechaty projects, write some documentation, or release your code as an open-source pluggable application. The ecosystem of pluggable applications is a big strength of Wechaty, help us
-  build it!
+- Contribute to open-source Wechaty projects, write some documentation, or release your code as an open-source pluggable application. The ecosystem of pluggable applications is a big strength of Wechaty, so help us build it!
```
Comment (lijiarui) [inline docusaurus/docs/contributing/overview.md:148]: Please add the following information:

```
## Know more about Wechaty

- [Wechaty Blog](https://wechaty.js.org/blog/)
- [Wechaty Contributors List](https://wechaty.js.org/contributors/): Get all of Wechaty contributor profiles/
- [Meeting Notes](https://docs.google.com/document/d/1fVCk8qRYc4RKGMf2UY5HOe07hEhPUOpGC34v88GEFJg/edit#heading=h.64c95c3y2l4v)： We use meeting notes to record all of the meetings
- [Branding Guides](https://wechaty.js.org/docs/marketing/branding/)： How to use Wechaty logo and other materials
- [YouTube](https://www.youtube.com/playlist?list=PL8hd9KDTdarDXf_Rxtr8meKhxtgcXMInh)： Wechaty video playlist.
- [Open Collective](https://opencollective.com/wechaty)： The only way to donate to Wechaty
- [Google Drive](https://drive.google.com/drive/folders/1KTnB3EOZo3nFRFSWoFc2-7LM7MgKQLzM)： Wechaty files
- [Photo Album](https://photos.google.com/share/AF1QipOWKUfUkjw-VzE0skrjmCwbwIWwuBiI7Li4UCbdXH62n8iH2ITnvDbPTsx4eBl8dw?key=cy1NdWFoUGpXanVmczVHSm84TVg1LXJWeW5HTDhR)： Wechaty photos
- [Hall of Fame](https://docs.google.com/document/d/1fVCk8qRYc4RKGMf2UY5HOe07hEhPUOpGC34v88GEFJg/edit#heading=h.64c95c3y2l4v): Know people in Wechaty
```

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Review by chris-4444 — COMMENTED** [review]
Included all your suggestions. Thanks @lijiarui 😊

**Review by lijiarui — APPROVED** [review]
Good job！ Cheers！

**Comment by lijiarui** [comment]
Please make GitHub action green then ：）

**Review by proudofsimin — APPROVED** [review]

**Review by nibble0101 — APPROVED** [review]
LGTM

**Pull request merged**